src/runtime/inspector/cdp_handler.cpp

```diff
@@ -215,3 +209,23 @@ void CdpHandler::addProtocolDefinitions(rapidjson::Value &domains, rapidjson::Do
     domains.PushBack(domainObj, allocator);
   }
 }
+
+string CdpHandler::extractDomain(const string &method)
```

Comment:
Because this PR adds a domain such as "JSAR.UniversalRenderServer" it might contain dots, please fix the method which extracts domain from the last dot, such as A.B....C.D => domain(A.B....C) and method(D)